spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -270,13 +271,7 @@ describe("CP/DP communication #" .. strategy .. " inc_sync=" .. inc_sync, functi
         method  = "GET",
         path    = "/soon-to-be-disabled",
       }))
-
-      if inc_sync == "on" then
-        -- XXX incremental sync does not skip_disabled_services by default
-        assert.res_status(200, res)
-      else
-        assert.res_status(404, res)
-      end
+      assert.res_status(404, res)
```

Comment:
Why?

Reply:
CP disabled service, so DP should return 404 to client.